contributors/distribution-guidelines.md

```diff
@@ -1,17 +1,42 @@
+How does this affect you?
 
 If you offer products or services related to Chef products, you may use the Chef name in describing and advertising your product so long as you don't mislead customers into thinking that either your website, service, or product is a Chef Software website, service, or product, or that Chef has any direct relationship with your organization. For example: "Nell's Management Dashboard for Chef Habitat" is okay. "Nell's Chef" is not okay, nor is "Nell's Chef Automate Dashboard".
 
-Additionally, commands used to execute the distribution must be different from those in Chef Software's distribution.  For example, if you create a distribution of Chef, the command to use your distribution's binary cannot be "chef", though it can be something like "kitty" - the goal is prevent confusion about which distributions come from Chef Software and which come from another source.
+Additionally, commands used to execute the distribution must be different from those in Chef Software's distribution. For example, if you create a distribution of Chef, the command to use your distribution's binary cannot be "chef", though it can be something like "kitty" - the goal is prevent confusion about which distributions come from Chef Software and which come from another source.
+
+## What are some examples?
+
+The Chef community has begun the process of creating a Community distribution of Chef itself - beginning with implementing a way to handle Chef’s trademarks throughout the code. They created a Chef::Dist namespace and replaced all Chef trademarks in the code with a configurable variable - i.e. “Chef Server” became “#{Chef::Dist::PRODUCT} Server”.
+
+This pull request was created in collaboration and reviewed by Chef Software and is a good implementation of what we are looking for when requiring that distros remove Chef’s trademarks. Please feel free to use it as a reference: https://github.com/chef/chef/pull/8368
+
+You can also see the conversation around the decisions of that pull request in this design proposal: https://github.com/chef/chef/issues/8376
+
+## What about Chef code libraries used in distributions?
+
+You do not have to remove Chef from the existing Ruby class names, e.g. Chef::Log. (However, derived or modified class names should not include Chef).
+
+## Do you need to change the installation paths for distributions?
+
+Yes, do not reuse /opt/chef, /etc/chef, etc. as installation paths. It is fine if the distribution uses symlinks or other tools to make migration easier.
+
+## Can you reference the Chef gem?
+
+Yes. Chef is currently removing binstubs from the Chef gem to allow it to function as a library, rather than an executable. It is fine to reference the Chef gem as a library from your code, but derived and modified class names should not include Chef trademarks.
```

Comment:
There's a pretty obvious contradiction, do not use /etc/chef, but you can use the chef gem and thus brings in the default path of /etc/chef for it's configuration path... It needs a review IMHO to cleat that up, as well as the cache folder.